consultorio-medico/src/main/java/com/iwa/iwaid/consultoriomedico/restcontrollers/PatientController.java

```diff
@@ -0,0 +1,58 @@
+package com.iwa.iwaid.consultoriomedico.restcontrollers;
+
+import com.iwa.iwaid.consultoriomedico.entity.PatientDTO;
+import com.iwa.iwaid.consultoriomedico.entity.PatientForm;
+import com.iwa.iwaid.consultoriomedico.services.PatientService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+
+@RestController
+@RequestMapping(path = "/iwaid/patients/")
+public class PatientController {
+
+    @Autowired
+    private final PatientService patientService;
+
+    public PatientController(PatientService patientService) {
+        this.patientService = patientService;
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<PatientDTO> getbyId(@PathVariable int id) {
+        Optional<PatientDTO> patient = Optional.ofNullable(patientService.getPatient(id));
+        if (patient.isPresent()) {
```

Comment:
mover esta validacion al service y considerar cambiar el not found por un error de Exception